cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -0,0 +1,56 @@
+Crontasks settings
+==================
+
+You find here, some good practices about crontasks management in Akeneo PIM Flexibility Edition :
+
+1. CRONs we provide by default uses a custom wrapper script defined by *SHELL* that call the PIM "bin/console" command.
+
+   It avoid us to write on each cron line the full script path to call and the redirection of logs.
+
+   Adding your own cron lines failes because it tries to run your command with our wrapper.
+   
+   To run your own script normally (without our wrapper), you just have to redefine the shell variable like this "SHELL=/bin/bash" before them
+    
+2. We would like to remind you that all servers are using UTC time (and so on for CRON).
+   
+   Think to convert you desired local time to UTC time.
+
+.. code-block:: bash
+
+    # ┌───────────── minute (0 - 59)
+    # │ ┌───────────── hour (0 - 23)
+    # │ │ ┌───────────── day of month (1 - 31)
+    # │ │ │ ┌───────────── month (1 - 12)
+    # │ │ │ │ ┌───────────── day of week (0 - 6) (Sunday to Saturday;
+    # │ │ │ │ │                                       7 is also Sunday on some systems)
+    # │ │ │ │ │
+    # │ │ │ │ │
+    # * * * * *  command to execute
+
+
+4. To be notify about CRONs executions (errors only), "MAILTO" variable should be changed and set with someone concerned of your company.
```

Comment:
notified

Reply:
should be changed.

Reply:
`4. To be notified about CRONs executions errors, you should change the default "MAILTO" variable value with someone concerned of your company.`